content/20.treatments.md

```diff
@@ -372,6 +373,49 @@ Additionally, one study identified an increased risk of mortality in older men r
 HCQ use for COVID-19 also leads to shortages for anti-malarial or anti-rheumatic use, where it has been definitively proven to be effective.
 Despite significant attention, these drugs appear to be ineffective against COVID-19.
 
+##### Dexamethasone
+
+Dexamethasone (9α-fluoro-16α-methylprednisolone) is a synthetic corticosteroid that binds to the glucocorticoid receptors [@doi:10.1021/cr068203e; @url:https://www.cebm.net/covid-19/dexamethasone/].
+It was first synthesized in the late 1950s as an anti-inflammatory and has been used to treat rheumatoid arthritis and other inflammatory conditions [@doi:10.1021/ja01545a061; @doi:10.1001/jama.1960.03030070009002].
+Steroids such as dexamethasone are widely available and affordable, and they are often used to treat community-acquired pneumonia [@doi:10.1038/s41392-020-0127-9].
+A clinical trial that began in 2012 recently reported that dexamethasone may improve outcomes for patients with ARDS [@doi:10/ggpxzc].
+However, meta-analysis of a small amount of available data regarding the use of dexamethasone to treat SARS suggested that it may, in fact, be associated with patient harm [@doi:10/ggks86], although these findings may have been biased by the fact that all of the studies examined were observational and a large number of inconclusive studies were not included [@doi:10/ggq356].
+Dexamethasone works as an anti-inflammatory by binding to glucocorticoid receptors with higher affinity than endogenous cortisol [@doi:10.1016/B978-0-323-48110-6.00034-X].
+In order to understand how dexamethasone works to reduce inflammation, it is necessary to consider the stress response more broadly.
+In response to stress, corticotropin‐releasing hormone stimulates the release of neurotransmitters known as catecholamines, such as epinephrine, and steroid hormones known as glucocorticoids, such as cortisol [@doi:10.1016/j.cyto.2015.01.008; @doi:10.1111/j.1749-6632.2002.tb04229.x].
+While catecholamines are often associated with the characteristic fight-or-flight response, the specific role that glucocorticoids play is less clear, although they are thought to be important to restoring homeostasis [@doi:10.1159/000054578].
+Immune challenge is a stressor that is known to interact closely with the stress response.
+Due to communication between the immune system and the central nervous system, macrophages can both respond to and produce catecholamines [@doi:10.1016/j.cyto.2015.01.008].
```

Comment:
Would you say that this is due to communication or would you say that this is a mechanism of communication? I am a bit thrown off by the wording here. A skim of that article makes me think this is a summary of a mechanism of how they can be linked.

Reply:
I think it's basically that the two systems are interwoven (which makes sense since it's all stress). So maybe:
```suggestion
The immune system can therefore interact with the central nervous system; for example, macrophages can both respond to and produce catecholamines [@doi:10.1016/j.cyto.2015.01.008].
```